src/bilder/images/edxapp_v2/templates/edxapp/mitx/common_values.yml.tmpl

```
@@ -116,6 +116,10 @@ CONTENTSTORE:
 
 ACTIVATION_EMAIL_SUPPORT_LINK: ''
 AFFILIATE_COOKIE_NAME: dev_affiliate_id
+ALLOWED_HOSTS:
+- https://{{ key "edxapp/lms-domain" }}
+- https://{{ key "edxapp/studio-domain" }}
+- https://{{ key "edxapp/preview-domain" }}
```

```suggestion
ALLOWED_HOSTS:
- {{ key "edxapp/lms-domain" }}
- {{ key "edxapp/studio-domain" }}
- {{ key "edxapp/preview-domain" }}
```
The HOST header is protocol agnostic so this just needs to be the fully qualified domain values.

---

This diff still needs to be applied